.github/workflows/docker.yml

```
@@ -99,25 +103,34 @@ jobs:
         run: |
           docker load --input ${{ runner.temp }}/base.tar
 
-      - name: Docker meta
-        id: meta
+      - name: Docker metadata for tag
+        if: startsWith(github.ref, 'refs/tags/')
+        id: meta_tag
         uses: docker/metadata-action@v5
         with:
           images: |
             ghcr.io/${{ github.repository_owner }}/aura-${{ matrix.binary-name }}
           tags: |
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}
-            type=ref,event=branch
-            type=sha
+            type=semver,pattern=v{{version}}
```

```suggestion
            type=semver,pattern={{version}}
```
nit: i suggest that we must tag our code with tags of `vMAJ.MIN.PATH` and not just `MAJ.MIN.PATCH`, and not rely on the workflow to add the letter "v".

---

{{version}} returns only the numeric value (e.g., 1.2.3) as documented [here](https://github.com/docker/metadata-action?tab=readme-ov-file#typesemver).